cypress/commons/actions/generic/ScenarioManager.js

```diff
@@ -46,6 +54,16 @@ function getScenarioValidationStatusLoadingSpinner(scenarioId) {
   return getScenarioAccordion(scenarioId).find(GENERIC_SELECTORS.scenario.validationStatusLoadingSpinner);
 }
 
+function getScenarioRunStatus(scenarioId, scenarioStatus, timeout = 5) {
+  return getScenarioAccordion(scenarioId).find(GENERIC_SELECTORS.scenario.scenarioStatus[scenarioStatus], {
+    timeout: timeout * 1000,
+  });
+}
+
+function getScenarioDataset(scenarioId) {
+  return getScenarioAccordion(scenarioId).find(GENERIC_SELECTORS.scenario.manager.scenarioDataset);
+}
+
```

Comment:
Same remark as in the other PR, I think we should use `datasets` (plural) in the selector name and function name.